### PR TITLE
Use php-ext-install for extensions not bundled by SymfonyCloud

### PR DESCRIPTION
**Pull request by tucksaun**

### CODE REVIEW

**Comment by tucksaun** [comment]
RFR: `php-ext-install` has been released

File: 00-flex.yaml [hunk 1]
```diff
@@ -46,6 +46,10 @@ template: |
             set -x -e
 
             curl -s https://get.symfony.com/cloud/configurator | (>&2 bash)
+            {{ range $ext := php_extensions -}}
+            {{- if not php_extension_available $ext $.PhpVersion -}}
+            (>&2 php-ext-install {{ $ext }} version)
```
Comment (fabpot) [inline 00-flex.yaml:51]: I would remove "version" here so that the command works out of the box without any changes (same below).

Comment (tucksaun) [reply 00-flex.yaml:51]: I won't as a version is required

Comment (fabpot) [reply 00-flex.yaml:51]: Let's comment the command then?

Comment (tucksaun) [reply 00-flex.yaml:51]: you mean like this:
```suggestion
            # (>&2 php-ext-install {{ $ext }} version)
```
?

Comment (tucksaun) [reply 00-flex.yaml:51]: or like this maybe:
```suggestion
            # (>&2 php-ext-install {{ $ext }} X.Y.Z)
```
?

Comment (fabpot) [reply 00-flex.yaml:51]: Second one

File: 01-silex.yaml [hunk 2]
```diff
@@ -46,6 +46,9 @@ template: |
             set -x -e
 
             curl -s https://get.symfony.com/cloud/configurator | (>&2 bash)
+            {{- if not php_extension_available $ext $.PhpVersion -}}
+            (>&2 php-ext-install {{ $ext }} version)
```
Comment (tucksaun) [inline 01-silex.yaml:50]:
```suggestion
            # (>&2 php-ext-install {{ $ext }} X.Y.Z)
```

**Pull request merged**